core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -49,6 +49,7 @@ object GameStarter {
         runAndMeasure("addCivilizations") {
             gameInfo.tileMap = tileMap
             tileMap.gameInfo = gameInfo // need to set this transient before placing units in the map
+            //gameInfo.ruleSet = ruleset doesn't work for some reason
```

Comment:
The reason is: this is early on in the construction of a card house on thin air - some things go first others come later, and in the middle you can't rely on anything existing unless you control the workflow such that it does. Anyway, you do not need to interfere here.

Reply:
thanks